internal/appsec/dyngo/instrumentation/httpinstr/http.go

```diff
@@ -60,50 +60,17 @@ func init() {
 // WrapHandler wraps the given HTTP handler with the abstract HTTP operation defined by HandlerOperationArgs and
 // HandlerOperationRes.
 func WrapHandler(handler http.Handler, span ddtrace.Span) http.Handler {
+	// TODO: move these to service entry tags
 	if !enabled {
 		span.SetTag("_dd.appsec.enabled", 0)
 		return handler
 	}
+	span.SetTag("_dd.appsec.enabled", 1)
+	span.SetTag("_dd.runtime_family", "go")
```

Comment:
This seems unrelated to the PR. Didn't we already do this elsewhere?

Reply:
It is related: this was in the removed lines below and moved out due to the refactoring with `MakeHandlerOperationArgs`.

As Kyle suggested in #1007, we should properly implement these via a new tracer configuration value for global service entry tags. I'll try to do it asap in a separate PR. Until then, this echo.v4 integration won't have those tags.